src/Raven.Server/Smuggler/Migration/Migrator_V3.cs

```diff
@@ -167,10 +173,10 @@ private async Task<string> MigrateRavenFs(string lastEtag)
                         WriteDocumentWithAttachment(documentActions, context, dataStream, key, metadata);
 
                         Parameters.Result.Documents.ReadCount++;
-                        if (Parameters.Result.Documents.Attachments.ReadCount % 50 == 0 || sp.ElapsedMilliseconds > 3000)
+                        if (Parameters.Result.Documents.ReadCount % 50 == 0 || sp.ElapsedMilliseconds > 3000)
```

Comment:
The attachment count needs to incremented

Reply:
:+1: